experimental/dds/tree/src/Identifiers.ts

```diff
@@ -24,7 +24,7 @@ export type UuidString = string & {
 export type AttributionId = UuidString;
 
 /**
- * A version 4, variant 2 uuid (https://datatracker.ietf.org/doc/html/rfc4122).
+ * A {@link https://datatracker.ietf.org/doc/html/rfc4122 | version 4, variant 2 uuid}.
```

Comment:
Same as https://github.com/microsoft/FluidFramework/pull/21768/files#r1664777966. This one doesn't actually go into public docs (that I could tell) because it's an experimental package, but might as well apply the better pattern for links.